### DISCUSSION TITLE
ERROR message opening navigator

### DISCUSSION THREAD

**Discussion by DebianGuru** [Security-Onion-Solutions/securityonion#13787] [2.4] [answered]
### Version

2.4.100

### Installation Method

Security Onion ISO image

### Description

other (please provide detail below)

### Installation Type

Standalone

### Location

on-prem with Internet access

### Hardware Specs

Exceeds minimum requirements

### CPU

20

### RAM

18GB

### Storage for /

166G

### Storage for /nsm

3.4TB

### Network Traffic Collection

span port

### Network Traffic Speeds

Less than 1Gbps

### Status

Yes, all services on all nodes are running OK

### Salt Status

No, there are no failures

### Logs

No, there are no additional clues

### Detail

When opening Navigator, I get a popup error saying, "ERROR parsing layer from assets/so/nav_layer_playbook.json, check the javascript console for more information." If I click "OK" I can continue to Navigator and it looks to me like things are working, but I've not dug into it far enough to confirm that everything IS actually working correctly.  I've tried this under 2 machine (both running the newest current release of Chrome).  Is this something to be concerned about?

### Guidelines

- [X] I have read the discussion guidelines at https://github.com/Security-Onion-Solutions/securityonion/discussions/1720 and assert that I have followed the guidelines.

**Comment by defensivedepth** [comment]
Try clearing your cache / inprivate window. Still happens?

**Reply by DebianGuru** [reply, marked as answer]
I appreciate your advice, but this problem has vanished on it' own.  I'm not sure what changed.
